docs/README.md

```diff
@@ -55,7 +55,7 @@ Modules for both sides:
 
 * [Coding style](development/coding-style.md)
 * [Source code directory structure](development/source-code-directory-structure.md)
-* [Technical differences to node-webkit](development/atom-shell-vs-node-webkit.md)
+* [Technical differences to NW.js](development/atom-shell-vs-nwjs.md)
```

Comment:
We should not change the URL, the link has been referenced by many articles and tweets.